client/.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/amuthamuhunthan/civicclimate0221/seechange/client/src/index.js":"1","/Users/amuthamuhunthan/civicclimate0221/seechange/client/src/App.js":"2"},{"size":172,"mtime":1613027656561,"results":"3","hashOfConfig":"4"},{"size":4601,"mtime":1613425693795,"results":"5","hashOfConfig":"4"},{"filePath":"6","messages":"7","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"bfp9b1",{"filePath":"8","messages":"9","errorCount":0,"warningCount":3,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"/Users/amuthamuhunthan/civicclimate0221/seechange/client/src/index.js",[],"/Users/amuthamuhunthan/civicclimate0221/seechange/client/src/App.js",["10","11","12"],{"ruleId":"13","severity":1,"message":"14","line":1,"column":17,"nodeType":"15","messageId":"16","endLine":1,"endColumn":25},{"ruleId":"13","severity":1,"message":"17","line":1,"column":27,"nodeType":"15","messageId":"16","endLine":1,"endColumn":36},{"ruleId":"18","severity":1,"message":"19","line":43,"column":5,"nodeType":"20","endLine":43,"endColumn":63},"no-unused-vars","'useState' is defined but never used.","Identifier","unusedVar","'useEffect' is defined but never used.","jsx-a11y/alt-text","img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.","JSXOpeningElement"]
```

Comment:
Hi! This file should be ignored by git. Feel free to add it to `.gitignore`, thanks!

Reply:
done

Reply:
Hey @Amutha37! I can still see this file in your PR. Probably you need to add `client/.eslintcache` as this is the path to this file? Try it out.

Reply:
path is added and done.

Reply:
Thanks for your PR @Amutha37 ! 
I've removed `client/.eslintcache` from PR, and updated `.gitignore` accordingly.

resolving this conversation.